incubator/fluentd/files/elasticsearch.conf

```diff
@@ -5,18 +5,18 @@
 
   flatten_hashes true
   flatten_hashes_separator _
-
-  buffer_type memory
-  buffer_chunk_limit {{ .Values.elasticsearch.buffer_chunk_limit }}
-  buffer_queue_limit {{ .Values.elasticsearch.buffer_queue_limit }}
-  flush_interval {{ .Values.elasticsearch.flush_interval }}
-  num_threads {{ .Values.elasticsearch.num_threads }}
-  disable_retry_limit
-  retry_wait 1s
-  max_retry_wait 60s
-  request_timeout {{ .Values.elasticsearch.request_timeout }}
-  buffer_queue_full_action {{ .Values.elasticsearch.buffer_queue_full_action }}
-
+  <buffer>
+    @type memory
+    chunk_limit_size {{ .Values.elasticsearch.chunk_limit_size }}
+    chunk_limit_records {{ .Values.elasticsearch.chunk_limit_records }}
+    flush_interval {{ .Values.elasticsearch.flush_interval }}
+    flush_thread_count {{ .Values.elasticsearch.flush_thread_count }}
+    retry_forever
+    retry_wait 1s
+    retry_max_interval 60s
+    retry_timeout {{ .Values.elasticsearch.retry_timeout }}
+    overflow_action {{ .Values.elasticsearch.buffer_queue_full_action }}
```

Comment:
I think this needs to be updated to `.Values.elasticsearch.elasticsearch.overflow_action` to line up with the values.yaml now, and with the updated parameter docs above

Reply:
that's a good catch!